portal.go

```diff
@@ -2060,6 +2069,116 @@ func (portal *Portal) convertWebPtoPNG(webpImage []byte) ([]byte, error) {
 	return pngBuffer.Bytes(), nil
 }
 
+func (portal *Portal) convertUrlPreview(source *waProto.ExtendedTextMessage) map[string]interface{} {
+	if ( source == nil ) {
+		return nil
+	}
+
+	matchedText := source.GetMatchedText()
+
+	if ( matchedText == "" ) {
```

Comment:
@tulir we need to get those pre-commit checks from #432

@adamvy parentheses shouldn't be used in `if` conditions.

Reply:
Whoops, my first go code :)

Fixed and ran go fmt which cleaned up some semicolons I put out of habit as well.